<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeDialog</class>
 <widget class="QDialog" name="RecipeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>530</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Main window</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Recipe Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Recipe Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="ingredientsLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>310</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Recipe ingredients</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionField">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>571</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameField">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="ingredientsField">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>571</width>
     <height>251</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <column>
    <property name="text">
     <string>name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>amount</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>unit</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addRowButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>340</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add ingredient</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>230</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear fields</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
